src/api/controllers/articleRatingController.js

```diff
@@ -0,0 +1,47 @@
+import model from '../models';
+import statuses from '../../helpers/constants/status.codes';
+import sendError from '../../helpers/error.sender';
+
+const { Ratings } = model;
+
+/**
+ *
+ *
+ * @class RatingsController
+ */
+class RatingsController {
+  /**
+   *this is the controller which enables the user to rate a certain article
+   *
+   * @author Frank Mutabazi
+   * @static
+   * @param {object} req - this is the request object
+   * @param {object} res - this is the response object
+   * @returns {object} - this function returns an object
+   * @memberof RatingsController
+   */
+  static async rateArticle(req, res) {
+    const { slug } = req.params;
+    const { rating } = req.body;
+    const { user: { id } } = req.user;
+
+    try {
+      await Ratings.create({
+        userId: id,
+        articleSlug: slug,
+        rating: parseInt(rating, 10)
+      });
+
+      return res.status(statuses.CREATED).json({
+        status: statuses.CREATED,
+        message: `you rated ${slug} with ${rating} rates, Thanks`
```

Comment:
@MCFrank16 Change this text to `Rating for ${slug} submitted successfully`

Reply:
yeah this is well structured